src/main/java/assets/javascript/jsterm.js

```diff
@@ -59,7 +59,7 @@ $(function () {
     }
 
 
-    $(window).keyup(function (event) {
+    $(window).keydown(function (event) {
```

Comment:
you have to use the keydown, because the backspace event from the browser fires on the key-down event, and by the time the key-up is fired, the browser already caught the back event.